### DISCUSSION TITLE
Request Ranking

### DISCUSSION THREAD

**Discussion by GuilhermeCouto** [dropwizard/dropwizard#9071] [Q&A] [answered]
Hi, is there a way to see which endpoints are requested the most? Link a ranking of endpoints?

**Comment by joschi** [comment, marked as answer]
You can enable access logs and run any log analyzer over them:

- https://www.dropwizard.io/en/release-4.0.x/manual/core.html#json-log-format
- https://www.dropwizard.io/en/release-4.0.x/manual/configuration.html#json-access-log-layout

**Reply by GuilhermeCouto** [reply]
Thank you!!
